experiments/Atmos_GCM/heldsuarez.jl

```diff
@@ -16,13 +16,15 @@ const p_ground = MSLP
 const T_initial = 255
 const domain_height = 30e3
 
-function init_heldsuarez!(state, aux, coords, t)
+function init_heldsuarez!(bl, state, aux, coords, t)
     global p_ground, T_initial
 
     FT = eltype(state)
 
+    # TODO: change to altitude function
     r = norm(coords, 2)
     h = r - FT(planet_radius)
+    # h = altitude(bl.orientation, aux)
```

Comment:
Need to add `using CLIMA.Atmos: altitude` for this to work. Or export `altitude` in `orientation.jl`.

Reply:
I was thinking that I'd do that in a separate PR